pkg/yttlibrary/json.go

```diff
@@ -37,6 +38,7 @@ func (b jsonModule) Encode(thread *starlark.Thread, f *starlark.Builtin, args st
 	}
 	allowedKWArgs := map[string]struct{}{
 		"indent": {},
+		"no_escape_html": {},
```

Comment:
parameter with name "escape_html" will be less confusing.
Its default value can be true, same as in json encoder.

Reply:
I would like too, but `core.BoolArg` return `false` by default:

https://github.com/carvel-dev/ytt/blob/b4d0ff06a83291341b06da166dfcb147baee1665/pkg/template/core/args.go#L12-L23

So it's impossible to say if a returned `false` value is explicitly requested by the user or is the default value, negate the value would not help neither.

The only way to implement it would be to either:
- re-wrote `core.BoolArg` to accept a default value.
- re-wrote `core.BoolArg` to return `nil` if the user doesn't pass a value, this would require to return a `*bool` instead of a `bool`.
- write another function `core.BoolArg` with a different signature to accept a default value... which obviously would be `true` because we already have one which return `false` by default, so maybe just provide another function which return `true` by default would be enough.

Which one of those solution do you prefer to be implemented? Or have you another idea on the topic?

Whatever your choice will be I'll be happy to implemented.

Reply:
During that time I'm implementing the first solution, which I think is best.

Reply:
Feel free to resolve this conversation if the implementation fit your expectations.

Reply:
I think changing `core.BoolArg` is fine.